notebooks/data-sources/TestGrid/metrics/probability_to_fail.ipynb

```diff
@@ -0,0 +1,1091 @@
+{
```

Comment:
<p>Do we need to retain <code>test_duration</code>  and <code>pass_or_fail</code>  here? Aren't they stored in other metric files? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/310/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
The `test_duration` we are using is different from all other files and `pass_or_fail` also we have added for the first time.

Reply:
<p>so the <code>pass_or_fail</code>  column was added to convert the boolean values to 1 or 0 for passing and failing tests respectively so that they can be used to calculate the probabilities. In the <code>test_pass_fialures.ipynb</code>  notebook, we are just retaining the boolean values as-is</p>

Reply:
Sure, but the translation from True/False to 1/0 is trivial. So I don't see why we need to store both a boolean and numerical representation of the same information in Ceph. Once you've calculated the metric of interest (probability to fail), what is the value of retaining columns with the interim values?

Reply:
<p>ah yes my bad, we can drop this column before saving it to ceph</p>